app/controllers/users/commencer_controller.rb

```diff
@@ -87,9 +87,7 @@ def procedure_for_help
       retrieve_procedure
     end
 
-    def nav_bar_profile
-      current_user ? :user : :guest
-    end
+    def nav_bar_profile = nav_bar_user_or_guest
```

Comment:
jouable ?
```suggestion
    # when instanciating manually the controller (see below),
    # we don't have request and current_user would fail
    def nav_bar_profile = request && current_user ? :user : :guest
```